login_bp.py

```diff
@@ -32,11 +33,12 @@ def redirect_back(endpoint, **values):
 
 
 class LoginForm(Form):
-    email = TextField('email', [validators.Required(), validators.Email(message='Chybný tvar emailové adresy.'),validators.Length(min=4)])
-    password = PasswordField('heslo', [validators.Required(), validators.Length(min=6, message = 'Heslo musí být alespoň 6 znaků dlouhé.')])
+    email = StringField('email', [validators.DataRequired(), validators.Email(message='Chybný tvar emailové adresy.'), validators.Length(min=4)])
+    password = PasswordField('heslo', [validators.DataRequired(), validators.Length(min=6, message = 'Heslo musí být alespoň 6 znaků dlouhé.')])
+
 
+blueprint = Blueprint('login_bp', __name__)
```

Comment:
You will have to adjust the blueprint name in a few places in layout.html, index.html and prihlaseni.html templates, too.